tubemq-core/src/main/java/com/tencent/tubemq/corebase/policies/FlowCtrlRuleHandler.java

```diff
@@ -439,7 +439,8 @@ public void clear() {
                 }
             }
         } catch (Throwable e2) {
-            throw new Exception("Parse flow-ctrl rule failure", e2);
+            throw new Exception(new StringBuilder(512).append("Parse flow-ctrl rule failure, ")
```

Comment:
Why do we prefer just throw exception with message of underneath exception instead of make use of stack tract mechanism?

Reply:
The error description in this class can clearly determine the cause of the problem.

Reply:
Thanks for your explanation.